app/views/network_router/new.html.haml

```diff
@@ -22,16 +22,21 @@
   %h3
     = _('Placement')
   .form-horizontal
+    .form-group{"ng-if" => "networkRouterModel.ems_id"}
```

Comment:
Probably wanted to add `ng-if` to the next line, instead of adding an empty `form-group`?